cloud/openstack/os_router.py

```diff
@@ -284,17 +305,32 @@ def main():
     if not HAS_SHADE:
         module.fail_json(msg='shade is required for this module')
 
+    if (module.params['project'] and
+            StrictVersion(shade.__version__) < StrictVersion('1.9.1')):
+        module.fail_json(msg="To utilize project, the installed version of"
+                             "the shade library MUST be >=1.9.1")
```

Comment:
You shouldn't make assumptions about the next shade release number. The current release is 1.9.0, so just check for <= 1.9.0.